### PR TITLE
fix(cli): add ability to escape at-prefixed parameter

### PR DESCRIPTION
**Pull request by pyhedgehog**
Fixes python-gitlab#2511

### CODE REVIEW

File: docs/cli-usage.rst [hunk 1]
```diff
@@ -305,6 +305,17 @@ command line. This is handy for values containing new lines for instance:
    EOF
    $ gitlab project create --name SuperProject --description @/tmp/description
 
+It you want to explicitly pass argument starting with "@" - double it:
```
Comment (nejch) [inline docs/cli-usage.rst:308]: Just a suggestion to be more explicit maybe:
```suggestion
It you want to explicitly pass an argument starting with ``@``,  you can escape it using ``@@``:
```

**Review by nejch — COMMENTED** [review]
Thanks a lot @pyhedgehog for this and for adding the tests. LGTM just a small suggestion for the docs.

**Comment by JohnVillalovos** [comment]
LGTM. @nejch I'll let you decide when to merge.

**Review by JohnVillalovos — APPROVED** [review]

**Review by nejch — APPROVED** [review]
Thanks @pyhedgehog and @JohnVillalovos!

**Pull request merged**